index.js

```diff
@@ -200,6 +200,16 @@ const _normalizeMatrixArray = float32Array => {
   }
 };
 
+const gamepadState = {
+  system: null,
+  isPresenting: true,
+}
+
+{
+  const vrContext = nativeVr.getContext();
+  gamepadState.system = nativeVr.VR_Init(nativeVr.EVRApplicationType.Scene);
```

Comment:
Won’t this auto boot vr for everything? If that’s true then I think this isn’t a good change in behavior.

Reply:
I’ll check. But if so, we can just return a stub OpenVR instance without actually calling init.

Reply:
I think it will boot immediately.

> But if so, we can just return a stub OpenVR instance without actually calling init.

Yes I think that's fine.